README.md

```diff
@@ -58,37 +58,35 @@ Supported Version: nginx 0.21.0/0.22.0
 ## Testing a sample
 
 1. Execute the below command to create a sample siddhi deployment.  
-`kubectl create -f ./deploy/crds/examples/example-http-default-app.yaml`
+`kubectl create -f ./deploy/crds/examples/example-stateless-log-app.yaml`
```

Comment:
let's move the examples to ./deploy/examples and the Siddhi CRD YAML to ./deploy directory since examples are not CRDs.